e2e/node/basic/identity.test.ts

```diff
@@ -26,6 +27,54 @@ function createSecpIdentity(seed: number): SignIdentity {
   return Secp256k1KeyIdentity.generate(new Uint8Array(seed1));
 }
 
+async function createEcdsaIdentity(): Promise<ECDSAKeyIdentity> {
+  const goldenSeed = {
+    key_ops: [],
+    ext: true,
+    kty: 'EC',
+    x: 'EaSQG1HdU7pMzMXaIjZmDGZCa2wit-JX95cuLjZXsZI',
+    y: '-z31VJQ1dNFRkg-eFdet9SPatYph0OPz5vbju0eeT6o',
+    crv: 'P-256',
+  };
+
+  const goldenPrivateKey = {
+    key_ops: [],
+    ext: true,
+    kty: 'EC',
+    x: 'wUmIVyFHanPKCknjOWlMFFr9OKSahY7p5yT1vn4D-kw',
+    y: 'KhYIS2VFq98PU08q1KGYidRfEJ2qV-EUrfaRQ4XbV_4',
+    crv: 'P-256',
+  };
+
+  const getTestKeyPair = async (): Promise<CryptoKeyPair> => {
```

Comment:
I should remove that - it was a holdover from me trying to coerce a CryptoKeyPair into existing, since you can't generate reproducibly from a seed. This function is not being used